src/utilities/ScrollToTop.jsx

```diff
@@ -0,0 +1,21 @@
+import { useEffect } from 'react';
+import { useLocation } from 'react-router-dom';
+
+const ScrollToTop = () => {
```

Comment:
If the component doesn't render anything, it would be better to make it as a hook